codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/wrappedlocalservice/extensions/DafnyPythonWrappedLocalServiceClientCodegenPlugin.java

```diff
@@ -41,6 +43,23 @@ public DafnyPythonWrappedLocalServiceClientCodegenPlugin(
     );
   }
 
+  /**
+   * Perform all transformations on the model before running wrapped localService codegen.
+   * @param model
+   * @param serviceShape
+   * @return
+   */
+  public static Model transformModelForWrappedLocalService(
+    Model model,
+    ServiceShape serviceShape
+  ) {
+    Model transformedModel = model;
+    transformedModel =
+      addWrappedLocalServiceTrait(transformedModel, serviceShape);
```

Comment:
off topic question: what does this transformation do exactly?

Reply:
Check out https://github.com/smithy-lang/smithy-dafny/blob/main-1.x/codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/wrappedlocalservice/extensions/DafnyPythonWrappedLocalServiceClientCodegenPlugin.java#L52

In preprocessing, it replaces a LocalServiceTrait on the ServiceTrait with a WrappedLocalServiceTrait.
This results in some different codegen.

Reply:
Ah thanks! I wasn't aware of that trait.